### PR TITLE
[ISSUE #2882] Remove the default request time of the plugin httpclient

### PR DESCRIPTION
**Pull request by zouchangfu**
image

### CODE REVIEW

**Comment by yu199195** [comment]
why remove this？  How do you control the timeout?

File: shenyu-spring-boot-starter/shenyu-spring-boot-starter-plugin/shenyu-spring-boot-starter-plugin-httpclient/src/main/java/org/apache/shenyu/springboot/starter/plugin/httpclient/HttpClientPluginConfiguration.java [hunk 1]
```diff
@@ -104,9 +103,9 @@ public HttpClient httpClient(final HttpClientProperties properties) {
                     }
                     // The write and read timeouts are serving as generic socket idle state handlers.
                     tcpClient = tcpClient.doOnConnected(connection -> {
+                        connection.addHandlerLast(new IdleStateHandler(properties.getReaderIdleTime(), properties.getWriterIdleTime(), properties.getAllIdleTime(), TimeUnit.MILLISECONDS));
```
Comment (yu199195) [inline shenyu-spring-boot-starter/shenyu-spring-boot-starter-plugin/shenyu-spring-boot-starter-plugin-httpclient/src/main/java/org/apache/shenyu/springboot/starter/plugin/httpclient/HttpClientPluginConfiguration.java:106]: why add this same？

Comment (zouchangfu) [reply shenyu-spring-boot-starter/shenyu-spring-boot-starter-plugin/shenyu-spring-boot-starter-plugin-httpclient/src/main/java/org/apache/shenyu/springboot/starter/plugin/httpclient/HttpClientPluginConfiguration.java:106]: I have removed the redundant code

Comment (dragon-zhang) [reply shenyu-spring-boot-starter/shenyu-spring-boot-starter-plugin/shenyu-spring-boot-starter-plugin-httpclient/src/main/java/org/apache/shenyu/springboot/starter/plugin/httpclient/HttpClientPluginConfiguration.java:106]: > why add this same？

He is trying to fix https://github.com/apache/incubator-shenyu/issues/2882. I think even if you remove this configuration, there should have another default configuration, @zouchangfu

Comment (dragon-zhang) [reply shenyu-spring-boot-starter/shenyu-spring-boot-starter-plugin/shenyu-spring-boot-starter-plugin-httpclient/src/main/java/org/apache/shenyu/springboot/starter/plugin/httpclient/HttpClientPluginConfiguration.java:106]: I found the default configuration in `org.springframework.http.client.Netty4ClientHttpRequestFactory#readTimeout`, is that correct ? @zouchangfu

Comment (dragon-zhang) [reply shenyu-spring-boot-starter/shenyu-spring-boot-starter-plugin/shenyu-spring-boot-starter-plugin-httpclient/src/main/java/org/apache/shenyu/springboot/starter/plugin/httpclient/HttpClientPluginConfiguration.java:106]: I don't think no limit is a good idea. Maybe it's better to limit the timeout that users configure too much ?

Comment (dragon-zhang) [reply shenyu-spring-boot-starter/shenyu-spring-boot-starter-plugin/shenyu-spring-boot-starter-plugin-httpclient/src/main/java/org/apache/shenyu/springboot/starter/plugin/httpclient/HttpClientPluginConfiguration.java:106]: Of course, also need to set a larger `readTimeout`.

**Pull request closed without merging**